versions/1.21.1/gradle.properties

```diff
@@ -17,23 +18,24 @@
 
 	# https://masa.dy.fi/maven/fi/dy/masa/malilib/
 	# https://jitpack.io/#sakura-ryoko/malilib
-	malilib_version = 1.21-0.21.1
+	# malilib-fabric-1.21-0.21.2.jar
+	malilib_version = 1.21-0.21.2
 
-	# https://legacy.curseforge.com/minecraft/mc-mods/tweakeroo/files
-	# tweakeroo-fabric-1.21.1-0.21.50.jar
-	tweakeroo_file_id = 5647797
+	# https://jitpack.io/#sakura-ryoko/tweakeroo
+	# tweakeroo-fabric-1.21-0.21.53.jar
+	tweakeroo_file_id = 6f2ad7a8e8
 
-	# https://legacy.curseforge.com/minecraft/mc-mods/item-scroller/files
-	# itemscroller-fabric-1.21.1-0.24.50.jar
-	itemscroller_file_id = 5647786
+	# https://jitpack.io/#sakura-ryoko/itemscroller
+	# itemscroller-fabric-1.21-0.24.53.jar
```

Comment:
Are these non-tweakeroo dependency updates necessary for this fix?

Reply:
Not really.  I primarily just wanted to see if any other dependencies broke at the same time.